### ISSUE TITLE
Error for Saving/Loading Multilabel Text Classification with Classweights

### ISSUE THREAD

**Issue by lady-vanessa** [deepset-ai/FARM#440]
**Question**
Hi there,
I use Farm 0.4.4 with Pytorch 1.5.0 and trained a multilabel text classification model with classweights. 
The training worked well, but unfortunately I connot load the model afterwards with `model = Inferencer.load(model_path)`

Error message:

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-3-08b27db6b27d> in <module>
     13 # Load model
     14 model_path = 'model/' + model_name
---> 15 model = Inferencer.load(model_path)

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/farm/infer.py in load(cls, model_name_or_path, batch_size, gpu, task_type, return_class_probs, strict, max_seq_len, doc_stride, extraction_layer, extraction_strategy, s3e_stats, num_processes, disable_tqdm)
    188         # a) either from local dir
    189         if os.path.exists(model_name_or_path):
--> 190             model = BaseAdaptiveModel.load(load_dir=model_name_or_path, device=device, strict=strict)
    191             if task_type == "embeddings":
    192                 processor = InferenceProcessor.load_from_dir(model_name_or_path)

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/farm/modeling/adaptive_model.py in load(cls, **kwargs)
     51             model = cls.subclasses["ONNXAdaptiveModel"].load(**kwargs)
     52         else:
---> 53             model = cls.subclasses["AdaptiveModel"].load(**kwargs)
     54         return model
     55 

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/farm/modeling/adaptive_model.py in load(cls, load_dir, device, strict, lm_name, processor)
    278         ph_output_type = []
    279         for config_file in ph_config_files:
--> 280             head = PredictionHead.load(config_file, strict=strict)
    281             prediction_heads.append(head)
    282             ph_output_type.append(head.ph_output_type)

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/farm/modeling/prediction_head.py in load(cls, config_file, strict, load_weights)
    115             model_file = cls._get_model_file(config_file=config_file)
    116             logger.info("Loading prediction head from {}".format(model_file))
--> 117             prediction_head.load_state_dict(torch.load(model_file, map_location=torch.device("cpu")), strict=strict)
    118         return prediction_head
    119 

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/torch/nn/modules/module.py in load_state_dict(self, state_dict, strict)
    828         if len(error_msgs) > 0:
    829             raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
--> 830                                self.__class__.__name__, "\n\t".join(error_msgs)))
    831         return _IncompatibleKeys(missing_keys, unexpected_keys)
    832 

RuntimeError: Error(s) in loading state_dict for MultiLabelTextClassificationHead:
	Unexpected key(s) in state_dict: "balanced_weights", "loss_fct.pos_weight". 
```

**Additional context**
To include class weights in my training, I used:

`prediction_head = MultiLabelTextClassificationHead(class_weights=data_silo.calculate_class_weights(task_name="text_classification"),num_labels=len(label_list))`

I had trained the same model without class_weights and had no problems loading it afterwards, so I think I am doing something wrong here.

**Comment by tholor** [comment]
Hey @lady-vanessa , 

Can you please try to install the latest version from the master branch?

This is related to https://github.com/deepset-ai/FARM/issues/428 which got fixed in #431 .

**Comment by lady-vanessa** [comment]
Hi @tholor ,
thanks for your quick answer!
Worked like a charm! :)
